api/db/database-builder/factory/build-passage.js

```diff
@@ -0,0 +1,21 @@
+import { databaseBuffer } from '../database-buffer.js';
+
+const buildPassage = function ({
+  id = databaseBuffer.getNextId(),
+  moduleId = 'mon super module',
```

Comment:
Nitpick : Vu que ça va être de l'uuid, peut-être mettre au moins avec des tirets

```suggestion
  moduleId = 'mon-super-module',
```

Reply:
Done